src/TorchSharp/TorchVision/Image.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+using static TorchSharp.torch;
+
+namespace TorchSharp.torchvision
+{
+    public static partial class io
+    {
+
+        public static IImager DefaultImager { get; set; } = new ImagerSharp();
+
+        public enum ImageFormat
```

Comment:
>ImageFormat [](http://example.com/codeflow?start=20&length=11)

nay reason these enums are defined inside the io class? do we expect these can be used in other scenarios?

Reply:
They should be in the same scope as they are in Pytorch, so that the qualified name is the same.